### PR TITLE
Fix incorrect value in RF

### PR DESCRIPTION
**Pull request by lordoz234**
Fixed incorrect value in plot_tree in Random Forest

### CODE REVIEW

**Review by Alexsandruss — CHANGES_REQUESTED** [review]
Look at Segmentation Fault in CircleCI

File: src/tree_visitor.h [hunk 1]
```diff
@@ -285,6 +285,20 @@ bool toSKLearnTreeObjectVisitor<M>::_onLeafNode(const typename TNVT<M>::leaf_des
 template<typename M>
 bool toSKLearnTreeObjectVisitor<M>::_onLeafNode(const typename TNVT<M>::leaf_desc_type  &desc, std::true_type)
 {
+    if (desc.level > 0)
+    {
+        size_t depth = desc.level - 1;
+        while (depth >= 0)
+        {
+            size_t id = parents[depth];
+            value_ar[id*1*class_count + desc.label] += desc.nNodeSampleCount;
```
Comment (agorshk) [inline src/tree_visitor.h:294]: Minor, as I see we don't have to multiply by 1

**Review by Alexsandruss — APPROVED** [review]

**Pull request merged**